src/main/java/com/fnz/TimeTracking/model/Utilisateur.java

```diff
@@ -23,10 +23,10 @@ public class Utilisateur {
     private String role;
     private String département;
     // Implement the getUsername and getPassword methods
-    //@Getter
-    //private String username; // Add a username field
-    //@Getter
-    //private String password; // Add a password field
+    @Getter
+    private String username; // Add a username field
```

Comment:
Why adding a username and a password ?

We are using a facial recongnition auth system.

a user will be identified by the photo field, which will be a face print